spec/features/user_signs_in_spec.rb

```diff
@@ -14,15 +14,15 @@
     fill_in 'Password', with: user.password
 
     click_button 'Log in'
-
-    expect(page).to have_content('Signed in successfully')
+    visit users_path
+    expect(page).to have_content('Welcome')
   end
 
   scenario 'specify invalid credentials' do
     visit new_user_session_path
 
     click_button 'Log in'
-    expect(page).to have_content('Invalid email or password')
+    expect(page).to have_content('Sign up')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.